### PR TITLE
Suggested Assignees

### PR DESCRIPTION
**Pull request by benibenj**

### CODE REVIEW

File: src/github/githubRepository.ts [hunk 1]
```diff
@@ -893,6 +894,51 @@ export class GitHubRepository implements vscode.Disposable {
...
+				query: schema.GetParticipants,
+				variables: {
+					owner: remote.owner,
+					name: remote.repositoryName,
+					number: pullRequestNumber,
+					first: 15
+				},
+			});
+
+			ret.push(
```
Comment (benibenj) [inline src/github/githubRepository.ts:914]: Currently I am not sorting the participants as GitHub might have a specific order for them, however, they don't mention an order in the API. Should I just order them Alphabetically?

File: src/github/githubRepository.ts [hunk 2]
```diff
@@ -893,6 +894,51 @@ export class GitHubRepository implements vscode.Disposable {
...
+				...result.data.repository.pullRequest.participants.nodes.map(node => {
+					return {
+						login: node.login,
+						avatarUrl: node.avatarUrl,
+						name: node.name,
+						url: node.url,
+						email: node.email,
+					};
+				}),
+			);
```
Comment (benibenj) [inline src/github/githubRepository.ts:924]: Currently I am not sorting the participants as GitHub might have a specific order for them, however, they don't mention an order in the API. Should I just order them Alphabetically?

File: src/github/folderRepositoryManager.ts [hunk 3]
```diff
@@ -724,6 +724,10 @@ export class FolderRepositoryManager implements vscode.Disposable {
 		return this._fetchAssignableUsersPromise;
 	}
 
+	async getPullRequestParticipants(githubRepository: GitHubRepository, pullRequestNumber: number): Promise<[IAccount[], IAccount]> {
```
Comment (alexr00) [inline src/github/folderRepositoryManager.ts:727]: I have a slight preference for returning an object like `{suggestions: IAccount[], viewer: IAccount}` as it is possible to see exactly what is being returned by looking at the method signature.

File: src/github/pullRequestOverview.ts [hunk 4]
```diff
@@ -391,7 +391,8 @@ export class PullRequestOverviewPanel extends IssueOverviewPanel<PullRequestMode
 	}
 	private getAssigneesQuickPickItems(
 		assignableUsers: IAccount[] | undefined,
-		suggestedReviewers: ISuggestedReviewer[] | undefined,
+		suggestedReviewers: IAccount[] | undefined,
```
Comment (alexr00) [inline src/github/pullRequestOverview.ts:394]: I think this is an old copy-paste error, but are there actually any callers of `getAssigneesQuickPickItems` that will pass undefined for these parameters now?

File: src/github/pullRequestOverview.ts [hunk 5]
```diff
@@ -402,31 +403,42 @@ export class PullRequestOverviewPanel extends IssueOverviewPanel<PullRequestMode
...
+		}
+
 		for (const suggestedReviewer of suggestedReviewers) {
-			const { login, name, isAuthor, isCommenter } = suggestedReviewer;
-			if (skipList.has(login)) {
+			if (skipList.has(suggestedReviewer.login)) {
 				continue;
 			}
 
-			const suggestionReason: string =
```
Comment (alexr00) [inline src/github/pullRequestOverview.ts:411]: Thanks for the code clean up!

**Review by alexr00 — CHANGES_REQUESTED** [review]
Change looks good! Just a few small style/preference/nit change requests.

File: src/github/githubRepository.ts [hunk 6]
```diff
@@ -893,6 +894,51 @@ export class GitHubRepository implements vscode.Disposable {
...
+		const ret: IAccount[] = [];
+
+		try {
+			const result: { data: PullRequestParticipantsResponse } = await query<PullRequestParticipantsResponse>({
+				query: schema.GetParticipants,
+				variables: {
+					owner: remote.owner,
+					name: remote.repositoryName,
+					number: pullRequestNumber,
+					first: 15
```
[the remark this answers is not in the document]
Comment (benibenj) [reply src/github/githubRepository.ts:910]: Should not have hardcoded the first option. Removed it and let the caller specify the first value.

File: src/github/pullRequestOverview.ts (hunk 4 above)
Comment (benibenj) [reply src/github/pullRequestOverview.ts:394]: You are right, I removed the undefined types

**Review by alexr00 — APPROVED** [review]
Looks great! I will branch for release tomorrow and then we can merge.

**Pull request merged**